usdt-impl/build.rs

```diff
@@ -27,6 +27,9 @@ enum Backend {
 
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
+    println!("cargo:rustc-check-cfg=cfg(usdt_backend_noop)");
+    println!("cargo:rustc-check-cfg=cfg(usdt_backend_linker)");
+    println!("cargo:rustc-check-cfg=cfg(usdt_backend_standard)");
```

Comment:
@pfmooney I didn't gate these here because I updated the pinned `rust-toolchain` as well, ensuring that a simple `cargo build` won't produce any warnings (which is what I was concerned about).

`cargo +1.75.0 build` will produce warnings, but I think that's okay (and it avoids the version-check dependency which we just removed :) )

Reply:
Fair enough